ToDoApp/ToDoApp/ToDoListView.swift

```diff
@@ -13,10 +13,11 @@ struct ToDoSection: Identifiable {
     let toDoItems: [TaskModel]
 }
 struct ToDoListView: View {
-    @StateObject var viewModel = ToDoListViewModel()
+    @StateObject var viewModel: ToDoListViewModel = ToDoListViewModel()
+    let addButttonSize = Constants.addButtonSize
```

Comment:
there is no need to have a separate constant on the ToDoListView, you can use Constants.addButtonSize directly